docs/xml/metainfo-desktopapp.xml

```diff
@@ -10,8 +10,8 @@
 	<section id="spec-appdata-introduction">
 		<title>Introduction</title>
 		<para>
-			A desktop application is an application which has a graphical user interface and is commonly used with mouse and keyboard. It also ships a Freedesktop
-			<filename>.desktop</filename> file to be visible in application menus.
+			A desktop application is interactive software that presents a graphical interface to the user.
+			To appear in application listings, the packaged application should include a <filename>.desktop</filename> file.
```

Comment:
I think I understand now that while the Freedesktop `.desktop` file is essential for processing/inclusion by AppStream, the metainfo files described in the following paragraph are optional extra information about distributed software.

As a result I wonder about two details here:

  * Perhaps we should indeed use the suggested word `must` rather than `should` (it's stricter terminology -- but for an audience already interested in `appstream`, that's probably what we want?).
  * It could be worth restoring the word `Freedesktop` to help people discover the [relevant format documentation](https://specifications.freedesktop.org/desktop-entry-spec/).  In fact, if the XSLT works OK with the nested XML content, let's try that as a hyperlink.

```suggestion
			To appear in application listings, the packaged application must include a <ulink url="https://specifications.freedesktop.org/desktop-entry-spec/latest/">Freedesktop <filename>.desktop</filename></ulink> file.
```

(this also makes me notice that some links elsewhere in the documentation reference a legacy/now-redirected `standards.freedesktop.org` domain)

Reply:
> (this also makes me notice that some links elsewhere in the documentation reference a legacy/now-redirected standards.freedesktop.org domain)

@jayaddison, if it would be of use, I'd accept changes across disparate files to fix that problem in this PR, but I imagine that the repository maintainers would prefer us to try that in a separate PR.

Would that be as simple as a F&R? It's indeed generally best to reduce the amount of 301s.

Reply:
> > (this also makes me notice that some links elsewhere in the documentation reference a legacy/now-redirected standards.freedesktop.org domain)
> 
> @jayaddison, if it would be of use, I'd accept changes across disparate files to fix that problem in this PR, but I imagine that the repository maintainers would prefer us to try that in a separate PR.

Agreed, if I were in the shoes of a maintainer, that would be my preference.
 
> Would that be as simple as a F&R? It's indeed generally best to reduce the amount of `301`s.

_Possibly_.  We should confirm that the URL path structure does not change -- or alternatively check for broken links before and after such a migration.

Reply:
URL-update suggestions opened as #663.